azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/controlbar/captions/CaptionsListView.kt

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.presentation.fragment.calling.controlbar.captions
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.widget.RelativeLayout
+import androidx.core.content.ContextCompat
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.lifecycleScope
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.azure.android.communication.ui.calling.implementation.R
+import com.azure.android.communication.ui.calling.utilities.BottomCellAdapter
+import com.azure.android.communication.ui.calling.utilities.BottomCellItem
+import com.azure.android.communication.ui.calling.utilities.BottomCellItemType
+import com.azure.android.communication.ui.calling.utilities.LocaleHelper
+import com.microsoft.fluentui.drawer.DrawerDialog
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+
+@SuppressLint("ViewConstructor")
+internal class CaptionsListView(
```

Comment:
CaptionsListView : RelativeLayout

Since ListView is a thing on android, looks off. Maybe CaptionsContainerView. 

Can hide the details if it's relative layout, frame layout, list view, recycler view, shouldn't matter. But avoid ListView since that's a thing on android.